com.zsmartsystems.zigbee.console.ember/src/main/java/com/zsmartsystems/zigbee/console/ember/EmberConsoleNcpRoutingCommand.java

```diff
@@ -88,48 +88,47 @@ public void process(ZigBeeNetworkManager networkManager, String[] args, PrintStr
         out.println("Discovery Table Size Config    : " + configDiscoveryTableSize);
         out.println("Route Table Size Config        : " + configRouteTableSize);
         out.println("Source Route Table Size Config : " + configSourceRouteTableSize);
-        out.println("Source Route Table Size        : " + sourceRouteFill + "/" + sourceRouteSize);
+        out.println("Source Route Table Size Usage  : " + sourceRouteFill + "/" + sourceRouteSize);
 
         out.println();
-        out.println("Destination   Next   Age    Concentrator   Status");
+        out.println("NCP Route Table :");
+        out.println();
+        out.println(String.format("%-15s  %-15s  %-6s  %-15s  %-15s", "Destination", "Next", "Age", "Concentrator", "Status"));
```

Comment:
Can we make these a constant since it's used in 2 places (for the title and the data)